dev-libs/libbase58/libbase58-0.1.4.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A library for encoding/decoding base58"
+DESCRIPTION="libbase58 provides encoding and decoding for the base58 format, \
+used in Bitcoin addresses."
+HOMEPAGE="https://github.com/bitcoin/libbase58"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libbase58/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="97e8c3387fd4ed90208a8101e658743efbb6e21714e594f0828e6414f92c0b1d"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
+	PATCHES="libbase58-$portVersion.patchset"
+fi
+
+PROVIDES="
+	libbase58$secondaryArchSuffix = $portVersion
+	lib:libbase58$secondaryArchSuffix = $portVersion
+"
+
+PROVIDES_devel="
+	devel:libbase58$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+"
+
+REQUIRES_devel="
+	libbase58$secondaryArchSuffix == $portVersion
+"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:autoreconf
+	cmd:aclocal
+	cmd:libtool
+"
+
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	prepareInstalledDevelLib libbase58
+	fixPkgconfig
+	packageEntries devel develop
```

Comment:
You shoud use $developDir instead of "develop".
So, if you keep support for x86_gcc2 primary arch, you should use:
```
INSTALL()
{
	make install
	prepareInstalledDevelLib libbase58
	fixPkgconfig

	if [ -n "$secondaryArchSuffix" ]; then
		rm -rf $documentationDir
		maybe_manDir_man3=
	else
		maybe_manDir_man3=$manDir/man3
	fi

	packageEntries devel \
		$developDir \
		$maybe_manDir_man3
}
```

In case you drop support for x86_gcc2 primary arch, then you would use
```
INSTALL()
{
	make install
	prepareInstalledDevelLib libbase58
	fixPkgconfig

	packageEntries devel \
		$developDir \
		$manDir/man3
}
```